src/rpc/misc.cpp

```diff
@@ -455,6 +455,25 @@ UniValue setmocktime(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+UniValue getmocktime(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0)
+        throw runtime_error(
+            "getmocktime\n"
+            "\nGet the current mocktime (-regtest only)\n"
```

Comment:
`setmocktime` checks for regtest mode and throws an exception otherwise. I don't believe it's wrong not to do that here, but the asymmetry of implementation here caught my eye.

Is it even worth considering an exception in this RPC command too? (not a blocker for this PR)

Reply:
hmm. i'm not sure :/ i think it's ok but lmk if you want me to remove it. thanks for the review.